src/tests/ftest/launch.py

```diff
@@ -1596,16 +1596,22 @@ def install_debuginfos():
         yum_args = ["--exclude", "ompi-debuginfo", "openmpi3"]
         if "suse" in distro_info.name.lower():
             yum_args.extend(["libpmemobj1", "python3"])
+            # There are no "debug" repos
+            # Packages are found within daos-stack* repos and opensuse* repos
+            cmds.append(
+                ["sudo", "dnf", "debuginfo-install", "--enablerepo='*'", "-y"] +
+                yum_args + ["daos-server", "gcc"])
         elif "centos" in distro_info.name.lower():
             yum_args.extend(["libpmemobj", "python36"])
+            cmds.append(
+                ["sudo", "debuginfo-install", "--enablerepo=*-debuginfo", "-y"] +
```

Comment:
On EL7, `debuginfo-install` already enables `*debug*` IIRC.

Reply:
Ok, let me confirm with and without --enablerepo argument just to confirm

Reply:
Debuginfo-install does not enable *debug* repos:

sudo debuginfo-install gcc
Loaded plugins: auto-update-debuginfo, fastestmirror
Loading mirror speeds from cached hostfile
 * extras: centos-distro.1gservers.com
extras                                                                                                                                                               | 2.9 kB  00:00:00
repo.dc.hpdd.intel.com_repository_centos-7.9-x86_64-group_                                                                                                           | 1.8 kB  00:00:00
repo.dc.hpdd.intel.com_repository_daos-stack-el-7-x86_64-stable-local_                                                                                               | 1.5 kB  00:00:00
Could not find debuginfo for main pkg: gcc-4.8.5-44.el7.x86_64
Could not find debuginfo pkg for dependency package libgcc-4.8.5-44.el7.x86_64
Could not find debuginfo pkg for dependency package libgomp-4.8.5-44.el7.x86_64
No debuginfo packages available to install


 → sudo debuginfo-install --enablerepo="*debuginfo" gcc
Loaded plugins: auto-update-debuginfo, fastestmirror
Loading mirror speeds from cached hostfile
epel-debuginfo/x86_64/metalink                                                                                                                         |  13 kB  00:00:00
 * epel-debuginfo: d2lzkl7pfhq30w.cloudfront.net
 * extras: centos-distro.1gservers.com
base-debuginfo                                                                                                                                         | 2.5 kB  00:00:00
extras                                                                                                                                                 | 2.9 kB  00:00:00
repo.dc.hpdd.intel.com_repository_centos-7.9-x86_64-group_                                                                                             | 1.8 kB  00:00:00
repo.dc.hpdd.intel.com_repository_daos-stack-el-7-x86_64-stable-local_                                                                                 | 1.5 kB  00:00:00
Package glibc-debuginfo-2.17-323.el7_9.x86_64 already installed and latest version
Package 1:gmp-debuginfo-6.0.0-15.el7.x86_64 already installed and latest version
Package libmpc-debuginfo-1.0.1-3.el7.x86_64 already installed and latest version
Package mpfr-debuginfo-3.1.1-4.el7.x86_64 already installed and latest version
Package zlib-debuginfo-1.2.7-18.el7.x86_64 already installed and latest version
Package yum-plugin-auto-update-debug-info-1.1.31-54.el7_8.noarch already installed and latest version
--> Running transaction check
---> Package gcc-debuginfo.x86_64 0:4.8.5-44.el7 will be installed
--> Processing Dependency: gcc-base-debuginfo = 4.8.5-44.el7 for package: gcc-debuginfo-4.8.5-44.el7.x86_64
--> Running transaction check
---> Package gcc-base-debuginfo.x86_64 0:4.8.5-44.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

Reply:
From the manpage:

>        debuginfo-install is a program which installs the RPMs needed to
       debug the specified package.  The package argument can be a
       wildcard, but will only match installed packages.  debuginfo-
       install will then enable any debuginfo repositories, and install
       the relevant debuginfo rpm.

From the source: https://github.com/rpm-software-management/yum-utils/blob/05db7ef501fc9d6698935bcc039c83c0761c3be2/debuginfo-install.py#L84-L99